project/ChatBuild.scala

```diff
@@ -44,8 +45,20 @@ object ChatBuild extends Build {
             .map((f1, f2) => Seq(f1.data, f2.data)),
         watchSources <++= (watchSources in frontend)
       )
+      .dependsOn(sharedJvm)
+
+  lazy val shared = (crossProject.crossType(CrossType.Pure) in file ("shared")).
+    settings(
+      scalaVersion:=scalaV,
+      libraryDependencies +="org.scala-js" %% "scalajs-stubs" % scalaJSVersion % "provided"
```

Comment:
Cool.  PS `libraryDependencies +="org.scala-js" %% "scalajs-stubs" % scalaJSVersion % "provided"` is useful if you ever need to use ScalaJS export annotations in the shared code, as I've run into [this issue](http://stackoverflow.com/questions/28413266/how-to-export-properties-of-shared-case-classes) before.  No need for it in this project yet, but just keep it in the back of your mind if you ever need it.